### PR TITLE
Install Server Foundation Components

### PR DESCRIPTION
**Pull request by zkayyali812**
Signed-off-by: Zachary Kayyali <zkayyali@redhat.com>

### CODE REVIEW

File: licensetest/license_test.go [hunk 1]
```diff
@@ -29,6 +30,9 @@ func TestLicense(t *testing.T) {
 			}
 			return nil
 		}
+		if strings.Contains(path, "testbin") {
```
Comment (JakobGray) [inline licensetest/license_test.go:33]: To exclude certain files from the license check, they can be included in the skip map like this
```
"../testbin": true, // Generated files
```

File: Makefile.dev [hunk 2]
```diff
@@ -3,4 +3,9 @@
 .PHONY: subscriptions
 
 subscriptions:
-	oc apply -k hack/subscriptions
\ No newline at end of file
+	oc apply -k hack/subscriptions
+
+add-images-local:
```
Comment (JakobGray) [inline Makefile.dev:8]: Could you add comments explaining what these targets/scripts do?

File: go.mod [hunk 3]
```diff
@@ -5,9 +5,50 @@ go 1.16
...
+	k8s.io/api v0.21.3
+	k8s.io/apimachinery v0.21.3
+	k8s.io/client-go v0.21.3
+	k8s.io/kube-aggregator v0.21.3
 	sigs.k8s.io/controller-runtime v0.8.3
 	sigs.k8s.io/yaml v1.2.0
+
 )
+
+// Pinned to k8s v0.20.0
```
Comment (JakobGray) [inline go.mod:18]: What is the reason for pinning to this k8s version?

**Review by JakobGray — CHANGES_REQUESTED** [review]

File: go.mod (hunk 3 above)
Comment (zkayyali812) [reply go.mod:18]: I used this k8s version as it is a recent version which is also compatible with the version of controller-runtime we are on. Also it is the one we use in MCH Operator repository.

This import is necessary for parsing certain resource types.

**Review by JakobGray — APPROVED** [review]

**Pull request merged**